providers/standard/src/airflow/providers/standard/operators/hitl.py

```diff
@@ -29,10 +29,10 @@
 from datetime import datetime, timezone
 from typing import TYPE_CHECKING, Any
 
-from airflow.models.baseoperator import BaseOperator
 from airflow.providers.standard.exceptions import HITLTimeoutError, HITLTriggerEventError
 from airflow.providers.standard.triggers.hitl import HITLTrigger, HITLTriggerEventSuccessPayload
 from airflow.providers.standard.utils.skipmixin import SkipMixin
+from airflow.sdk.bases.operator import BaseOperator
```

Comment:
This could break compat, you should add it with a try / except here.

Unless the operator is only supposed to work for 3.1+ 

cc @Lee-W

Reply:
It is for 3.1 only, see a few lines above:
```python
if not AIRFLOW_V_3_1_PLUS:
    raise AirflowOptionalProviderFeatureException("Human in the loop functionality needs Airflow 3.1+.")
```

This was changed to the below anyway
```python
from airflow.providers.standard.version_compat import BaseOperator
```

Reply:
Yeah checked with Wei too, sounds good.